src/index.css

```diff
@@ -61,7 +60,7 @@ blockquote {
   overflow-wrap: break-word;
   margin: 30px 0 33px 0;
   /* text-align: center; */
```

Comment:
I guess we can remove this line too, I haven't seen it

Reply:
Okay I'll get rid of them